outlook/indico_outlook/plugin.py

```diff
@@ -66,35 +66,51 @@ class OutlookUserPreferences(ExtraUserPreferences):
                                                                                preserve_data=True)],
                                       choices=_status_choices,
                                       description=_('The status for Outlook Calendar entries')),
-        'outlook_status_overrides': MultipleItemsField(
-            _('Outlook entry status overrides'),
+        'outlook_reminder': BooleanField(_('Reminder'), [HiddenUnless('extra_outlook_active',
+                                                preserve_data=True)],
+                                    description=_('Enable calendar reminder')),
+        'outlook_reminder_minutes': IntegerField(_('Reminder time'), [HiddenUnless('extra_outlook_active',
+                                                preserve_data=True), NumberRange(min=0)],
+                                    description=_('Remind users X minutes before the event')),
+        'outlook_overrides': MultipleItemsField(
+            _('Outlook entry overrides'),
             [HiddenUnless('extra_outlook_active', preserve_data=True)],
```

Comment:
Actually, you could add a custom validator here to check for useless entries and reject them

Reply:
I tried that, but that only works when you click save and not as you edit entries, which is nicer. Is that possible?

Reply:
nope

Reply:
I've side-stepped this issue by making all fields mandatory.